test/mocks/PriceSourceMock.sol

```diff
@@ -19,4 +19,14 @@ contract PriceSourceMock {
         decimals = _decimals;
     }
 
+    function latestRoundData() external view returns (
+        uint80 roundId,
+        int256 answer,
+        uint256 startedAt,
+        uint256 updatedAt,
+        uint80 answeredInRound
+    ) {
+        return (0, latestAnswer, 0, 0, 0);
+    }
+
 }
```

Comment:
A new line could be added at the end since the file is edited

Reply:
done, added to the other file too